lib/analyzer.js

```diff
@@ -279,6 +287,9 @@ Analyzer.prototype._processScript = function _processScript(script, href) {
     parsedJs = jsParse(script.childNodes[0].value, this.attachAST);
     if (parsedJs.elements) {
       parsedJs.elements.forEach(function(element) {
+        if(this.attachAST) {
+          element.script = script;
```

Comment:
Maybe use `scriptElement` instead? I might expect `script` to be the text content.